vasyl_varyshchuk/Week_1/task_3/generate_file_size.py

```diff
@@ -0,0 +1,55 @@
+"""
+This module returns random size of file
+"""
+
+import random
+
+list_of_files = ['reports.txt', 'monthly_reports.zip', 'README.txt',
+                 'picture.jpg', 'logo.ttf', 'config.ini']
+
+
+def recognize_file_type(file_name):
+    _, extension = file_name.split('.')
+    return extension
+
+
+def txt_handler():
+    return random.randint(100, 450)
+
+
+def zip_handler():
+    return random.randint(500, 700)
+
+
+def jpg_handler():
+    return random.randint(800, 1000)
+
+
+def ttf_handler():
+    return random.randint(1100, 1300)
+
+
+def ini_handler():
+        return random.randint(1300, 1500)
+
+
+functions = {
+    'txt': txt_handler,
+    'zip': zip_handler,
+    'jpg': jpg_handler,
+    'ttf': ttf_handler,
+    'ini': ini_handler
+ }
+
+
+def run_executors(file_name):
```

Comment:
As I can see from the task description, we expect to execute our function in this way:

     run_executors(list_of_files)

As I can see from your code, it's working in another way.
Could you please fix it?

Reply:
I fixed it by using for loop over list items. It looks simple, but to honest, I am not sure this best decision for this case. It is interesting to know your opinion.